src/main/java/com/brightpath/learnify/controller/WorkspaceController.java

```diff
@@ -33,7 +34,8 @@ public ResponseEntity<List<WorkspaceSummaryDto>> listWorkspaces() {
 
     @Override
     public ResponseEntity<WorkspaceSummaryDto> createWorkspace(WorkspaceCreateDto workspaceCreateDto) {
-        Workspace workspace = workspaceService.createWorkspace(workspaceCreateDto.getDisplayName());
+        String userId = userIdentityService.getCurrentUserId();
```

Comment:
shouldn't it be placed in service?

Reply:
Maybe it could be but since this is reletad to the http request (we are still getting the user token from the request header under the hood) I think the rest controller is the good place for that